aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -221,8 +247,6 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle) : AztecFormat
                 editableText.removeSpan(it)
             }
         }
-        // remove the CSS style span
-        removeInlineCssStyle()
```

Comment:
Suggest we keep this for now since it does not affect background color, see comment below https://github.com/wordpress-mobile/AztecEditor-Android/issues/914

Reply:
ok, if really doesn't affect bg color i think we can keep, but its something to take a look in the future, that method doesn't make much sense